content/en/platform/corda/5.0/key-concepts/fundamentals/ledger/transactions/_index.md

```diff
@@ -0,0 +1,146 @@
+* Unique — no other committed transaction has consumed any of the inputs to the proposed transaction. Uniqueness is determined by a [notary](#notary).
+
+If the transaction gathers all of the required signatures without meeting these conditions, the transaction’s outputs are not valid and will not be accepted as inputs to subsequent transactions.
+
+## Reference States
+Some states need to be referred to by the contracts of other input or output states but not updated/consumed. These are known as reference states. When a state is added to the references list of a transaction, instead of the inputs or outputs list, it is treated as a reference state. There are two important differences between regular input states and reference states:
+* The specified notary for the transaction does check whether the reference states are current. However, reference states are not consumed when the transaction containing them is committed to the ledger.
+* The contracts for reference states are not executed for the transaction containing them.
+
+## Other Transaction Components
+As well as input states and output states, transactions contain:
+
+* [Commands](#commands)
+* [Attachments](#attachments)
+* [Time windows](#time-windows)
+* [A notary](#notary)
+
+{{< note >}}
+Attachments will be implemented in a future release.
+{{< /note >}}
+
+For example, suppose Alice uses $5 cash to pay off $5 of an IOU with Bob. This transaction contains a settlement command which reduces the amount outstanding on the IOU, and a payment command which changes the ownership of $5 from Alice to Bob. It also has two supporting attachments, and is notarized by `NotaryClusterA` if the notary pool receives it within the specified time window:
+
+{{< 
+  figure
+	 width="50%"
+	 src="full-tx.png"
+	 figcaption="Transaction Backchain Example"
+>}}
+
+### Commands
+Commands enable you to make adjustments to rules. Including a command in a transaction indicates the transaction’s intent, affecting how you check the validity of the transaction. For example, if you had a transaction with a cash state and a bond state as inputs, and a cash state and a bond state as outputs, it could represent either a bond purchase or a coupon payment on a bond. Different rules are required to define what constitutes a valid transaction depending on whether it is a purchase or a coupon payment. For example, in the case of a purchase, you would require a change in the bond’s current owner, but not for a coupon payment.
+
+{{< 
+  figure
+	 width="50%"
+	 src="commands.png"
+	 figcaption="Commands"
+>}}
+
+### Time Windows
+
+Each transaction requires a defined time window during which it can be approved and completed. The minimum requirement is an end date by which the transaction validation and notarisation must be complete. You may also want a proposed transaction to be approved during a certain period of time. For example:
```

Comment:
```suggestion
Each transaction requires a defined time window during which it can be approved and completed. The minimum requirement is an end date by which the transaction validation and notarization must be complete. You may also want a proposed transaction to be approved during a certain period of time. For example:
```